### PR TITLE
Tank drive

### PR DESCRIPTION
**Pull request by JacksonMcLaughlin**

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<classpath>
```
Comment (octylFractal) [inline .classpath]: don't we need this file?

File: src/org/usfirst/frc/team7171/robot/OI.java [hunk 2]
```diff
@@ -36,4 +37,11 @@
 	// Start the command when the button is released and let it run the command
 	// until it is finished as determined by it's isFinished method.
 	// button.whenReleased(new ExampleCommand());
+	public static Joystick jsFwBack;
+	public static Joystick jsTurn;
+	
+	public OI() {
+		jsFwBack = new Joystick(RobotMap.JS_FW_BACK);
```
Comment (octylFractal) [inline src/org/usfirst/frc/team7171/robot/OI.java]: don't set static fields in a constructor, thanks

File: src/org/usfirst/frc/team7171/robot/subsystems/DriveTrainSide.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package org.usfirst.frc.team7171.robot.subsystems;
+
+import org.usfirst.frc.team7171.robot.RobotMap;
+
+import com.ctre.CANTalon;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class DriveTrainSide extends Subsystem {
```
Comment (octylFractal) [inline src/org/usfirst/frc/team7171/robot/subsystems/DriveTrainSide.java]: does this need to be a subsystem?

**Review by octylFractal — CHANGES_REQUESTED** [review]
some changes maybe?

File: .classpath (hunk 1 above)
Comment (JacksonMcLaughlin) [reply .classpath]: its not in rouge-cephalopod. because gradle?

File: src/org/usfirst/frc/team7171/robot/subsystems/DriveTrainSide.java (hunk 3 above)
Comment (JacksonMcLaughlin) [reply src/org/usfirst/frc/team7171/robot/subsystems/DriveTrainSide.java]: no

**Review by octylFractal — APPROVED** [review]

**Pull request merged**